core/frontend/src/render/webgl/Target.ts

```diff
@@ -593,19 +593,17 @@ export abstract class Target extends RenderTarget implements RenderTargetDebugCo
     }
   }
 
-  private _frameStatsCollector = new FrameStatsCollector();
+  private _frameStatsCollector?: FrameStatsCollector;

```

Comment:
Just a thought, if you leave this like it was previously you won't have to check for `undefined` every time you access it.

Reply:
You make a good point. I had considered that and for some reason went with this approach, but I am not a fan of the undefined checks... I'll see what I can do.

Reply:
It is a bit more complicated now because both Viewport and Target need access to the collector.

Reply:
I'm saying: keep assignFrameStats and make it overwrite the one you initially created on the Target. Similar to how we default-initialize _hilites then overwrite it later.

Reply:
Yep, we're saying the same thing. I agree - it's a better pattern. Will push shortly.

Reply:
I pushed this change - can you mark 'resolved' if looks good?